experimental/ixdtf/src/parser.rs

```diff
@@ -0,0 +1,506 @@
+#[derive(Clone, Copy)]
+enum  DateTimeSeparator {
+    CapitalT,
+    LowCaseT,
+    Space,
+}
+
+impl DateTimeSeparator {
+    fn value(&self) -> u8 {
+        match *self {
+            DateTimeSeparator::CapitalT => 84,
```

Comment:
Suggestion, here and elsewhere: use `b''` syntax, like `b'T'`, instead of ASCII code constants in code